elasticsearch/src/main/fabric8/deployment.yml

```diff
@@ -0,0 +1,27 @@
+spec:
+  replicas: ${replicas}
+  template:
+    spec:
+      containers:
+      - name: elasticsearch
+        image: ${fabric8.dockerUser}elasticsearch-k8s:${elasticsearch.version}
+        env:
+          - name: SERVICE
+            value: elasticsearch-masters
+        volumeMounts:
+          - name: data
+            mountPath: /usr/share/elasticsearch/data
+        livenessProbe:
+          tcpSocket:
+            port: 9300
+          initialDelaySeconds: 30
+        readinessProbe:
+          httpGet:
+            port: 9200
+            path: /_template/logstash
+          initialDelaySeconds: 5
+      - name: logstash-template
+        image: ${fabric8.dockerUser}elasticsearch-logstash-template:${project.version}
```

Comment:
FYI: I'm going to make the registry more transparent, so that we can avoid this nasty `${fabric8.dockerUser}` hack. This will probably done by an `RegistryEnricher` which post-processes the generated files and then, according to the given configuration, adds a registry and/or user.